components/ambient-cli/demo-github.sh

```
@@ -0,0 +1,421 @@
+sep
+
+# ── gather inputs ──────────────────────────────────────────────────────────────
+
+announce "0 · Configuration"
+
+# GitHub PAT
+if [[ -n "${GITHUB_TOKEN_FILE:-}" ]]; then
+    dim "   Using GITHUB_TOKEN_FILE=${GITHUB_TOKEN_FILE}"
+    GITHUB_PAT_PATH="${GITHUB_TOKEN_FILE}"
+else
+    DEFAULT_TOKEN_PATH="${GITHUB_TOKEN:-}"
```

Shellcheck (0.11.0)

[warning] DEFAULT_TOKEN_PATH appears unused. Verify use (or export if used externally).

(SC2034)